source/filesystem.c

```diff
@@ -196,6 +196,17 @@ void addExecutableToMenu(menu_s* m, char* execPath)
 	addMenuEntryCopy(m, &tmpEntry);
 }
 
+void loadDescriptorNetloaderApp(descriptor_s* d, char* execPath) {
+	static char xmlPath[128];
+	snprintf(xmlPath, 128, "%s", execPath);
+	int l = strlen(xmlPath);
+	xmlPath[l-1] = 0;
+	xmlPath[l-2] = 'l';
+	xmlPath[l-3] = 'm';
+	xmlPath[l-4] = 'x';
```

Comment:
The file is always a .3dsx, it's the same code in the part when the launcher load the 3dsx, and check if the .xml exists ;)